### DISCUSSION TITLE
Trying to add tags to pages and failing (newbie).

### DISCUSSION THREAD

**Discussion by efcsn** [automadcms/discussions#12] [Q&A] [answered]
I am reaching out for assistance with a specific aspect of using Automad. As a complete novice in the world of web creation, I am struggling to understand how to create “tags” on my pages and configure them correctly.

I have attempted to follow the user guide, but unfortunately, I am still unclear about how to proceed. Specifically, I am unsure about how to configure the “Url Tag Link Target” feature.

Would it be possible for someone with more experience to provide guidance or assistance in this matter? I would greatly appreciate any help you can offer.

By the way I'm using Automad version 1.10.9 in a _yunohost_ installation.

**Comment by marcantondahmen** [comment, marked as answer]
Hi there and welcome here! 

Tags in general are a way to group pages. Pages that share common tags are considered "related".

The way tags are implemented is fully dependent on the theme in use.

In case of the standard theme you might want to take a look at the theme readme. A link to the readme can be found in the "General Data and Files" page of the dashboard directly below the theme dropdown.

Basically you want to create a page where you showcase all projects or blog posts you have. Such a page should either have the "Blog" or "Portfolio" template assigned. The slug of that page, for example `/blog`, can be used in the `Url Tag Link Target`. Then when you assign a tag to a page and someone clicks on the tag label below the title of that page, the visitor will be directed to `/blog` and the displayed page cards are automatically filtered by the clicked tag.

Also related pages are by default displayed below the main content of a page.

**Reply by efcsn** [reply]
Oh, you have shown me the light! Despite reading several times the readme I couldn't see _the way_.

Note to mention that the option "show all pages in Pagelist" must be enabled in the "general data" config to show the "tagged" pages.

Thank you.

Edit: in my despair I've even updated to the 2.0.0-alpha.5 version. It's nicer.

**Reply by marcantondahmen** [reply]
You're welcome! Happy to help!
